Course 13 - GitHub!/Course13Notes.js

```diff
@@ -1,52 +1,97 @@
-// What is GitHub?
-// I don't know lol
-
-// git repository
-// is a virtual storage of your project. It allows you to save versions of your code, which you can access when needed. REWORD THIS
-// Executing this command will create a new .git subdirectory in your current working directory. This will also create a new master branch. REWORD THIS
-
-/* What is a branch?
-No clue fml
-local
-remote
-origin - 
-*/
+/* GitHub Notes
+
+1. Log in to Github and click on the repository you wish to clone
+
+https://github.com/partnerhero/Programming-101
+
+2. Click the green Code button to reveal the URL and click the clipboard icon to copy it
+
+3. Navigate to your desired local project folder in command prompt
+    -the cloning process will create a new folder with your project already inside
 
-/* Useful Commands
+4. git clone <URL>
+    -this puts all the files on your computer // AND goes through the init process so you don't have to do git init beforehand
 
-git init - creates a new repo (repository)
+5. git checkout -b <new branch name>
+    -this creates a new branch and moves you to that branch
+        -changes that you make to your code on this branch will be separate from the master branch
 
-git clone - copies an existing repository while creating a new repository. No need to use git init if a repository already exists. git clone goes through the initialization process for you.
+6. git push -u origin <your new branch name>
+    -this creates a remote branch on github that you can push your changes to
 
-git status - shows you the status of any changes you've made
+7. Make your changes
+    -create a folder with your name on it and add an index.js file that logs something to the console
 
-git branch - shows the available brnahces and highlights to branch you're on
+8. git status once you've saved your changes
+    -this will show you any files that haven't been added or committed yet
 
-git pull - updates your local directory with any new changes????
+9. git add .
+    -this adds all your save changes so you can commit them
 
-git add - adds your changes to something?
-git add . - adds all changes
+10. git commit -m "<brief message here>"
+    -this commits your changes to your local branch
 
-git commit - adds your added changes to something else lol? local branch?
-git commit -m "Message here" - commits changes with a note about the changes
+11. git push
+    -pushes your changes to the remote branch 
+    -can also use git push origin <remote branch name>
 
-git push - pushes commited changes to remote branch
-git push -u - maybe??? why???
```

Comment:
Review the syntax for this command and update accordingly

Reply:
@znegomir The "lols" and others were removed with the latest commit. The [files changed](https://github.com/partnerhero/Programming-101/pull/9/files) has some filters you can use to show changes from "all commits"